src/service/gov_notify_service.js

```diff
@@ -7,10 +7,19 @@ export function setClient(client) {
 }
 
 async function sendEmail(emailAddress, personalisation, template) {
-  await _notifyClient.sendEmail(template, emailAddress, {
-    personalisation: personalisation,
-    reference: 'test-find-grant-reference',
-  });
+  try {
+    await _notifyClient.sendEmail(template, emailAddress, {
+      personalisation: personalisation,
+      reference: 'test-find-grant-reference',
+    });
+  } catch (error) {
+    if(error.response.data) {
+      console.error(error.response.data, { depth: null });
```

Comment:
afaik the `depth: null` trick only works with `console.dir` which accepts a single object to log and an options object as arguments. the other `console` functions all expect to receive a variable number of arguments, all of which will be logged.

Reply:
<img width="619" alt="Screenshot 2023-07-13 at 12 49 09" src="https://github.com/cabinetoffice/gap-find-web/assets/101722961/f2820389-c289-423b-887e-c8899fae48e4">
seems to log the errors here at least

Reply:
aye might just not be nested enough actually - updating to console.dir